lib/bundle.js

```diff
@@ -175,7 +176,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     });
   }
 
-  return bundle()
+  return bundle(systemBuilder)
```

Comment:
Perhaps make this `return bundle(new Builder())` and remove the `systemBuilder` reference entirely at the top?